libs/api/src/main/java/de/cyface/api/MongoAuthHandler.java

```diff
@@ -1,121 +0,0 @@
-/*
- * Copyright 2019-2021 Cyface GmbH
- *
- * This file is part of the Cyface Data Collector.
- *
- * The Cyface Data Collector is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * The Cyface Data Collector is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with the Cyface Data Collector. If not, see <http://www.gnu.org/licenses/>.
- */
-package de.cyface.api;
-
-import org.apache.commons.lang3.Validate;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import io.vertx.core.Handler;
-import io.vertx.core.http.HttpHeaders;
-import io.vertx.ext.auth.mongo.MongoAuthentication;
-import io.vertx.ext.web.RoutingContext;
-
-/**
- * Abstract base class for all requests to a collector endpoint. This class makes sure that such requests are properly
- * authorized.
- *
- * @author Klemens Muthmann
- * @author Armin Schnabel
- * @version 3.0.0
- * @since 1.0.0
- */
-public final class MongoAuthHandler implements Handler<RoutingContext> {
```

Comment:
This class was duplicate with as `Authorizer` which the Collector now also uses, so this can go.